bin/crew

```diff
@@ -1703,6 +1711,9 @@ def archive_package(crew_archive_dest)
     end
   end
   system "sha256sum #{crew_archive_dest}/#{pkg_name} > #{crew_archive_dest}/#{pkg_name}.sha256"
+  # Copy package file for the successfully generated package to CREW_LOCAL_REPO_ROOT .
+  FileUtils.cp "#{CREW_PACKAGES_PATH}/#{@pkgName}.rb", "#{CREW_LOCAL_REPO_ROOT}/packages/"
```

Comment:
Shouldn't this be  `FileUtils.cp "#{CREW_LOCAL_REPO_ROOT}/packages/#{@pkgName}.rb", CREW_PACKAGES_PATH`?

Reply:
Let me know if you need this behind a flag... otherwise this does look correct...